### PR TITLE
Improved instructions for the use of HA Cloud

### PR DESCRIPTION
**Pull request by astaves**
Minor fixes and improved instructions to aid the use of Host Access for Cloud

### CODE REVIEW

File: tutorial/gettingstarted/eclipse/README.md [hunk 1]
```diff
@@ -612,6 +612,16 @@ To execute the JCL, you need to run the application in an instance of the Micro
 
 **Important:** You need Enterprise Developer or Enterprise Developer for z Systems to execute the application as running applications is not supported in Enterprise Developer Connect.
 
+Before you proceed, ensure that the default settings are applied to the Directory Sever:
+
+1.  Click the **Start** menu and open the **Services** application. Navigate to Micro Focus Directory Server to view its status and set it to **Running** if it is not already started.
```
Comment (amy-johnstone) [inline tutorial/gettingstarted/eclipse/README.md]: Unbold "Start" and "Services".

File: tutorial/gettingstarted/visualstudio/readme.md [hunk 2]
```diff
@@ -680,14 +681,24 @@ This sample provides a PowerShell script that creates the region definition to u
 
     This executes the script and creates the Enterprise Server region definition file, **BANKDEMO.xml**, in the same folder. The file is configured for the location in which you have saved the sample files.
 
-To import the definition of the Bankdemo logical server (LSER) in Enterprise Server:
+Ensure that the default settings are applied to the Directory Sever:
 
-1.  In Visual Studio, open the Server Explorer window.
+1.  Click the **Start** menu and open the **Services** application. Navigate to Micro Focus Directory Server to view its status and set it to **Running** if it is not already started.
```
Comment (amy-johnstone) [inline tutorial/gettingstarted/visualstudio/readme.md]: Unbold "Start" and "Services".

**Review by amy-johnstone — CHANGES_REQUESTED** [review]
Only a minor change about referring to the Start menu and Services application.

File: tutorial/gettingstarted/eclipse/README.md (hunk 1 above)
Comment (astaves) [reply tutorial/gettingstarted/eclipse/README.md]: I've now made that change

File: tutorial/gettingstarted/visualstudio/readme.md (hunk 2 above)
Comment (astaves) [reply tutorial/gettingstarted/visualstudio/readme.md]: I've now made that change

**Pull request merged**